src/MoBi.Assets/MoBi.Assets.csproj

```diff
@@ -26,9 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="12.0.335" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.335" />
-    <PackageReference Include="OSPSuite.Core" Version="12.0.335" />
+    <PackageReference Include="OSPSuite.Assets" Version="12.0.0-OGKUq" />
+    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.0-OGKUq" />
+    <PackageReference Include="OSPSuite.Core" Version="12.0.0-OGKUq" />
```

Comment:
core with the "in children" extension was merged, so now we can replace this with that version